pxr/usdImaging/bin/testusdview/CMakeLists.txt

```diff
@@ -331,9 +432,253 @@ if (NOT PXR_HEADLESS_TEST_MODE)
         EXPECTED_RETURN_CODE 0
     )
 
+    pxr_register_test(testUsdviewFreeCamera
```

Comment:
Just realized we should disable all of the image diff tests on macOS until we get better imaging support on that platform (soon!).